2_delivery_care.do

```diff
@@ -173,7 +178,7 @@ c_sba_eff2_q
 			if inlist(country_name,"Congodr2017") {	
 				global mn19 "mn19a mn19c mn19d"
 			}
-			if inlist(country_name,"Togo2017") {	
+			if inlist(country_name,"Togo2017","Tonga2019") {	
 				global mn19 "mn19a mn19b mn19c mn19d"
```

Comment:
Simply for future reference, the four categories recognised are Medical doctor, Nurse/Midwife, Health Officer and Community Nurse.